models/readme.md

```diff
@@ -47,7 +55,7 @@ benchit benchmark linear.py --search-dir selftest
 You can also evaluate an entire corpus with a command like this:
 ```
 cd MLAGILITY_ROOT/models # MLAGILITY_ROOT is where you cloned mlagility
-benchit benchmark --all --search-dir selftest
+benchit models/selftest/*.py
```

Comment:
You're missing at least one instance of `--all` and `--search-dir` in this document. Make sure to search the entire repo and eliminate all references to both of those flags.

Reply:
Good catch!